src/libraries/System.Runtime.Serialization.Schema/tests/System/Runtime/Serialization/Schema/SchemaUtils.cs

```diff
@@ -222,10 +237,10 @@ internal class SchemaUtils
         #endregion
 
 
-        internal static XsdDataContractImporter CreateImporterWithDefaultOptions()
+        internal static XsdDataContractImporter CreateImporterWithOptions(ImportOptions opts = null)
         {
             XsdDataContractImporter importer = new XsdDataContractImporter();
-            importer.Options = new ImportOptions();
+            importer.Options ??= new ImportOptions();
```

Comment:
You didn't use the passed in opts. I think you mean this:
```suggestion
            importer.Options = opts ?? new ImportOptions();
```